Formula/php.rb

```diff
@@ -102,12 +106,24 @@ def install
     ENV["lt_cv_path_SED"] = "sed"
 
     # system pkg-config missing
-    ENV["SASL_CFLAGS"] = "-I#{MacOS.sdk_path_if_needed}/usr/include/sasl"
-    ENV["SASL_LIBS"] = "-lsasl2"
+    ENV["KERBEROS_CFLAGS"] = " "
+    on_macos do
+      # Ensure that libxml2 will be detected correctly in older MacOS
```

Comment:
```suggestion
```

Seems to be a copy paste from above - this block doesn't refer to either libxml2 or older macOS.